libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -45,14 +45,31 @@ func setupIPv6BridgeNetFiltering(config *networkConfiguration, _ *bridgeInterfac
 	return nil
 }
 
+func LoadBridgeModules() error {
+	moduleMap := map[string]string {
+		"arp_probe":                "bridge",
+		"bridge-nf-call-iptables":  "br_netfilter",
+	}
+
+	for fPath, modName := range moduleMap {
+		fullPath := "/proc/sys/net/bridge/" + fPath
+		if _, err := os.Stat(fullPath); err != nil {
```

Comment:
It might be worth logging here ... then we'll be able to tell if dockerd attempted the load - and we'll be more likely to spot the problem if we come across a host where the test for "bridge-nf-call-iptables" doesn't work as expected, causing us to run unnecessary modprobes.

```suggestion
		log.G(context.TODO()).Info("Loading br_netfilter")
		if _, err := os.Stat(fullPath); err != nil {
```